inc/pkthlp.h

```diff
@@ -308,6 +309,129 @@ PktBuildTcpFrame(
     return TRUE;
 }
 
+//
+// Build a QUIC frame from the given parameters and paylaod.
+// The frame is meant to be wraped in a TCP or UDP frames,
+// as the payload in `PktBuildTcpFrame` or `PktBuildUdpFrame`.
+//
+inline
+_Success_(return != FALSE)
+BOOLEAN
+PktBuildQuicLongHeader(
+    _Out_writes_bytes_(*BufferSize) VOID *Buffer,
+    _Inout_ UINT32 *BufferSize,
+    _In_ UINT8 TypeAndSpecificBits,
+    _In_reads_bytes_(DestConnIdLength) CONST UCHAR *DestConnId,
+    _In_ UINT8 DestConnIdLength,
+    _In_reads_bytes_(SrcConnIdLength) CONST UCHAR *SrcConnId,
+    _In_ UINT8 SrcConnIdLength,
+    _In_opt_ CONST UCHAR *Payload,
+    _In_ UINT16 PayloadLength
+    )
+{
+    UINT8* Cursor = (UINT8*)Buffer;
+    CONST UINT32 Version = 1;
```

Comment:
If this is hard coded, please change the name to `PktBuildQuicV1LongHeader`

Reply:
Will do + add a version parameter to the main entry point function (asserting it is 1) to make other versions easy to support in the future.